### DISCUSSION TITLE
Problem with fuel prediction page [Non-priority please do other more important things first]

### DISCUSSION THREAD

**Discussion by AirbusFlyer** [flybywiresim/aircraft#2337] [Help]
It seems that this happens when you enter a flight plan with a length of over the aircraft range:
<img width="173" alt="Why" src="https://user-images.githubusercontent.com/57779477/101698424-4bd3a400-3a2e-11eb-96a3-05179b62ae80.PNG">
I discovered this when I flew from New Mexico (24N) to Belfast (EGAC) and had a route with a length of 4002NM. I found out that (at least currently this I hope will eventually be fixed) the aircraft can fly up to around 5500-6100NM. I discovered this through some very limited testing at various altitudes and taking a look at the fuel consumption. Through this, I found at higher altitudes you typically use more fuel and that the speed gain from higher altitudes is not worth the extra fuel. This is how I knew I would have enough fuel to make that flight without stopping for fuel. If this could eventually be fixed that would be great. It is not a big deal for me as I typically use Simbrief anyway.

**Comment by Benjozork** [comment]
Hello,

What the commenter meant in #2328 was to open an issue, using the "Issues" tab. You opened another discussion, which is not handled as a bug report. Please make sure to follow the given template in the new issue.
